ocs_ci/templates/lvmo-deployment/lvm-cluster-default.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: odf-lvmcluster
+  namespace: openshift-storage
```

Comment:
I am not sure the openshift-storage NS exists after fresh OCP deployment. As in ODF deployment it's created when doing UI deployment, in CLI deployment, we need to create this NS manually.

Reply:
lvm cluster must be created after lvm operator (and openshift-storage namespace)